src/pages/Pools/Home/ClosurePrompts.tsx

```diff
@@ -43,11 +44,11 @@ export const ClosurePrompts = () => {
     memberCounter === '1';
 
   // depositor needs to unbond funds
-  const depositorCanUnbond = active.toNumber() > 0 && !targets.length;
+  const depositorCanUnbond = !active.gt(new BN('0')) && !targets.length;
```

Comment:
Why did you flip the condition here? It used to check whether the `active` is > 0 but now the first part is true only if `active` is 0?